charts/vector/values.yaml

```diff
@@ -81,9 +81,14 @@ podDisruptionBudget:
   maxUnavailable: 1
 
 rbac:
-  # rbac.create -- If true, create and use RBAC resources
+  # rbac.create -- If true, create and use RBAC resources. Only valid for the Agent role
```

Comment:
We currently only render rbac resources for Agents as only the `kubernetes_logs` source requires access to the kube api